book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './rental'
+
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    rent_book = Rental.new(date, self, person)
+    @rentals.push(rent_book)
+  end
```

Comment:
- [ ] Kindly remove this unnecessary **line 14** which is `@rentals.push(rent_book)` from **add_rental** method to make your code clean :blush: Second it will automatically push your rental into the rentals array so you don't need to do that here manually :+1: